tests/Go/TestUtils/JoinPointsExtractor.php

```diff
@@ -0,0 +1,116 @@
+<?php
+/*
+ * Go! AOP framework
+ *
+ * @copyright Copyright 2011, Lisachenko Alexander <lisachenko.it@gmail.com>
+ *
+ * This source file is subject to the license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Go\TestUtils;
+
+use PhpParser\Node;
+use PhpParser\NodeTraverser;
+use PhpParser\Node\Stmt;
+use PhpParser\NodeVisitorAbstract;
+use PhpParser\Parser;
+use PhpParser\ParserFactory;
+use PhpParser\PrettyPrinter\Standard as Printer;
+
+/**
+ * Extracts join points definitions from woven class proxy by parsing and traversing class AST.
+ */
+final class JoinPointsExtractor extends NodeVisitorAbstract
```

Comment:
That was my first guess, then I introspected implementation of class `Go\ParserReflection\ReflectionProperty` because I needed that one. 

However - not that tests are executed in one thread (PHPUnit), commands (different scope) in other. They do not share settings, they do not share class loader (Composer and AOP proxied composer).

That means that `Go\ParserReflection\ReflectionEngine` does not have in context of PHPUnit idea how to locate Proxy class in command scope.